experiment/quick-sort/simulation/partition.html

```diff
@@ -36,11 +36,14 @@
        
        <div id="legend-comment-wrapper">
           <ul class="legend">
-            <li><span class="left"></span> Left Pointer </li>
-            <li><span class="right"></span> Right Pointer </li>
-            <li><span class="merge"></span> Merge of Pointers</li>
-            <li><span class="selected-pivot"></span> Selected Pivot Element</li>
-            <li><span class="swapping"></span> Swapping</li>
+            <table>
+              <tr><td><b>Legend:</b></td></tr>
+              <tr><td><span class="left"></span>&ensp;Left Pointer </td></tr>
+              <tr><td><span class="right"></span>&ensp;Right Pointer </td></tr>
```

Comment:
Separate out the colour and its corresponding text into two <td> cells for much clarity, wont need ensp then. Remove the span and instead give one td element the required bg colour.